rubrik_cdm/data_management.py

```diff
@@ -1620,3 +1620,60 @@ def vcenter_refresh_vm(self, vm_name, timeout=15):  # pylint: ignore
         config = {'vmMoid':moid}
         self.log("vcenter_refresh_vm: Refreshing vSphere VM {} metadata.".format(vm_name))
         self.post('internal', '/vmware/vcenter/{}/refresh_vm'.format(vcenter_id), config, timeout)
+
+    def get_vsphere_vm(self, timeout=15, **query):  # pylint: ignore
+        """Get summary of all the VMs.
+
+        Keyword Arguments:
+            name {str} -- Search by using a virtual machine name.
```

Comment:
`Keyword Arguments` should have a default value. `Arguments` are the correct way of to describe these (except for `timeout`)